make/mcu/AT32F4.mk

```diff
@@ -2,5 +2,57 @@
 # AT32F4 Make file include
 #
 
+STDPERIPH_DIR   = $(ROOT)/lib/main/AT32F43x/drivers
+STDPERIPH_SRC   = \
+            $(notdir $(wildcard $(STDPERIPH_DIR)/src/*.c)) \
+
+EXCLUDES        = at32f435_437_dvp.c \
+				  at32f435_437_can.c \
+				  at32f435_437_xmc.c \
+				  at32f435_437_emac
```

Comment:
tabs. I see this is just copy/pasting from another file, but can you fix it, please?

Reply:
Fixed